services/storage/src/simcore_service_storage/datcore_dsm.py

```diff
@@ -58,7 +58,32 @@ async def list_files(
         )
 
     async def get_file(self, user_id: UserID, file_id: StorageFileID) -> FileMetaData:
-        raise NotImplementedError
+        api_token, api_secret = await self._get_datcore_tokens(user_id)
+
+        package_files = await datcore_adapter.get_package_files(
+            self.app, api_token, api_secret, file_id
+        )
+        assert (
+            len(package_files) == 0
```

Comment:
I think this is wrong, right? if you expect no files then `package_files[0]["content"]` would raise.


NOTE: Note that all the `assert` will disappear from the code in production (we include a special variable that "disable" all `assert` in the code). We use `assert` just to "document" some conditions or invariants on the code
If you want really to raise an exception you have to use
```python
if not package_files:
   raise MyError("More than one file in package, this breaks the current assumption")
```
If instead you just wanted to "document" the code, then it fine but please add `# nosec` at the end
```python
assert package_files  # nosec
```